modules/backing-up-red-hat-quay.adoc

```diff
@@ -0,0 +1,320 @@
+[[backing-up-red-hat-quay]]
+== Backing up Red Hat Quay
+
+This procedure is exclusively for OpenShift Container Platform and NooBaa deployments.
+
+.Prerequisites
+
+* A {productname} deployment on OpenShift Container Platform.
+
+
+.Procedure
+
+
+. Backup the `QuayRegistry` custom resource by exporting it:
++
+[source,terminal]
+----
+$ oc get quayregistry <quay-registry-name> -n <quay-registry-namespace> -o yaml > quay-registry.yaml
+----
+
+. Edit the resulting `quayregistry.yaml` and remove the status section and the following metadata fields:
++
+[source,yaml]
+----
+  metadata.creationTimestamp
+  metadata.finalizers
+  metadata.generation
+  metadata.resourceVersion
+  metadata.uid
+----
+
+. Backup the managed keys secret:
++
+[NOTE]
+====
+If you are running a version older than Red Hat Quay 3.7.0, this step can be skipped. Some secrets are automatically generated while deploying Quay for the first time. These are stored in a secret called `<quayname>quay-registry-managed-secret-keys` in the QuayRegistry namespace.
+====
```

Comment:
In order to be consistent with the line 41,  the secret is better to be `<quay-registry-name>-quay-registry-managed-secret-keys `, not `<quayname>quay-registry-managed-secret-keys`